debian/postinst

```diff
@@ -12,7 +12,8 @@ case "$1" in
     configure)
 
         # Create custom sudoers file
-        echo "%sudo   ALL=(root) NOPASSWD: /usr/bin/kano-launcher" > $TMP_FILE
+	echo "Defaults env_keep += KLOG_FORCE_FLUSH" > $TMP_FILE
```

Comment:
There is something wrong with the indentation of this line - have we got a mix of tabs and spaces?